src/components/ManageUsers/UserSearch.jsx

```diff
@@ -0,0 +1,23 @@
+const UserSearch = () => {
+  return (
+    <div className="usersearch__container">
```

Comment:
here the same with classes
```suggestion
    <div className="user-search__container">
```